data/dictionary.yaml

```diff
@@ -578,6 +585,13 @@
     @[Decoders](Decoder), in the computational redstone sense. For example
     serial hex to parallel binary is a deserializer encoder.
 
+- term: Short Pulse
+  tags:
+    - Concept
+  description: >-
+    A pulse of 2@[gt](gametick) or shorter, often used to make sticky pistons
+    drop their blocks (see also: @[Togglestate]). Opposite of a @[Long pulse].
```

Comment:
> often used to make sticky pistons
    drop their blocks
   Not accurate as often imo. Short pulses are more common in wiring with observers when the length of the pulse doesn't matter. Many storagetech contraptions use them not because of the block dropping property, but for convenience of wiring. It can also be mentioned in a general way of saying "power it with an observer" (or repeater, or something else)

Reply:
Proposed version;
```
A pulse of 2@[gt](gametick) or shorter, usually caused by an observer or a monostable circuit (link). It is often used to transfer single signals or encoded data (link encoded tech). It can also be used to make sticky pistons drop their blocks ((see also: @[Togglestate]).
```

Reply:
I'd argue the specific term "short pulse" might be more often used to refer specifically to the piston behavior, despite *short pulses* being used plenty for other stuff too?